modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -385,26 +388,58 @@ def value_counts(self, **kwargs):
         return self.__constructor__(new_frame, shape_hint="column")
 
     def _get_index(self):
+        """
+        Return frame's index.
+
+        Returns
+        -------
+        pandas.Index
+        """
         if self._modin_frame._has_unsupported_data:
             return default_axis_getter(0)(self)
         return self._modin_frame.index
 
     def _set_index(self, index):
+        """
+        Set new index.
+
+        Parameters
+        ----------
+        index : pandas.Index
+            A new index.
+        """
         if self._modin_frame._has_unsupported_data:
-            return default_axis_setter(0)(self, index)
-        default_axis_setter(0)(self, index)
-        # NotImplementedError: OmnisciOnRayFrame._set_index is not yet suported
-        # self._modin_frame.index = index
+            default_axis_setter(0)(self, index)
+        else:
+            default_axis_setter(0)(self, index)
+            # NotImplementedError: OmnisciOnRayFrame._set_index is not yet suported
+            # self._modin_frame.index = index
 
     def _get_columns(self):
+        """
+        Return frame's columns.
+
+        Returns
+        -------
+        pandas.Index
+        """
         if self._modin_frame._has_unsupported_data:
             return default_axis_getter(1)(self)
         return self._modin_frame.columns
 
     def _set_columns(self, columns):
+        """
+        Set new columns.
+
+        Parameters
+        ----------
+        columns : list-like
+            New columns.
+        """
         if self._modin_frame._has_unsupported_data:
-            return default_axis_setter(1)(self, columns)
-        self._modin_frame = self._modin_frame._set_columns(columns)
+            default_axis_setter(1)(self, columns)
+        else:
+            self._modin_frame = self._modin_frame._set_columns(columns)
```

Comment:
This change doesn't seem to relate to the issue.

Reply:
This change is also to pass the doc checker.

Reply:
ok